### DISCUSSION TITLE
Call for Mentors for Open Source Promotion Plan 2023!

### DISCUSSION THREAD

**Discussion by nobodyiam** [apolloconfig/apollo#4815] [General]
@apolloconfig/committers @apolloconfig/members 

Apollo community has signed up for [Open Source Promotion Plan](https://summer-ospp.ac.cn/), which encourages university students to participate in the development and maintenance of open-source software. Students can apply for the project they are interested in and guided by the senior maintainer (community mentor) once they are selected.

This is a great opportunity to cultivate fresh blood in the open-source community, and I strongly encourage committers and members to sign up for the mentor and contribute tasks for the students to participate.
If you want to be a mentor but don't have a task in your mind, then you may use choose from the following candidates:
* #3505 
* #4316 
* #4708

Here is the timeline:

| **Date** | **Event** |
| ----------- | ---------------------- |
| 03/06-03/30 | Community Application and Review |
| 03/31 | Community Announced |
| 04/01-04/03 | Committee Confirms the Supported Project Quantity and Notifies Community |
| 04/04-04/26 | Project Release |
| 04/26-04/28 | Committee Review |
| 04/29 | Project Announced |
| 04/29-06/04 | Student Registration, Communication, and Project Application Submission |
| 06/05-06/25 | Project Application Review |
| 06/26 | Accepted Student Projects Announced |
| 06/26-06/30 | Community Bonding |
| 07/01-09/30 | Coding and Development |
| 10/01-10/31 | Evaluations – Mentor Final Term Review and PR/MR Merge |
| 11/01-11/08 | Evaluations – Committee Final Term Review |
| 11/09 | Results Announced |
| November | Outstanding Students Selection |

**Comment by klboke** [comment]
> https://github.com/apolloconfig/apollo/issues/4316

+1

**Comment by Anilople** [comment]
https://github.com/apolloconfig/apollo/issues/3505

+1

**Comment by JaredTan95** [comment]
https://github.com/apolloconfig/apollo/issues/4708

+1
